closure/testing/BUILD

```diff
@@ -22,6 +22,10 @@ licenses(["notice"])  # Apache 2.0
 load("//closure:defs.bzl", "closure_js_binary", "closure_js_library")
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 
+# html template used by webdriver_test.
+exports_files(["gen_webtest_html.template"])
```

Comment:
I  don't think this is needed. The file is not used outside of this package, right ?

Reply:
This file is used in the `_gen_test_html` rule in webdriver_test.bzl.  It will complain if I don't exports_files it here.